### DISCUSSION TITLE
profile

### DISCUSSION THREAD

**Discussion by niko3332** [community/community#18328] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi! How often are achievements updated in profile?

Yesterday I did a merge of branches, but a new achievement (YOLO) has not yet appeared

**Comment by weixiabing** [comment, marked as answer]
soon

**Comment by HUMBLEGROUND** [comment]
Maybe this can help https://github.com/Schweinepriester/github-profile-achievements

**Comment by Nishant1500** [comment]
It is not yet known to us that how much time it takes to update the achievements. Some badges updates immediately and some are delayed.

These answers are also promising:
https://github.com/github-community/community/discussions/18335#discussioncomment-2927554
